static/js/app.js

```diff
@@ -276,10 +276,54 @@ $(function() {
             return params;
         },
         setMultipleMarkersIcon: function() {
+            var markerGroups = [];
+            var markerObj;
+            var groupID = 1;
+            var groupsColours = [];
             _.each(this.oms.markersNearAnyOtherMarker(), function(marker) {
                 marker.icon = getIcon("multiple", SEVERITY_VARIOUS);
                 marker.title = 'מספר תאונות בנקודה זו';
+
+                markerObj = {
```

Comment:
It would be nicer to have markerGroups be an associative array (an object) where the keys are the marker IDs and the values are the group IDs. This way you don't need to create a "marker object" and you can look up markers by their ID simply with bracket notation.